sources/engine/Stride.Assets.Models/ImportModelCommand.Lod.cs

```diff
@@ -31,18 +31,13 @@ private object ExportLOD(ICommandContext commandContext, ContentManager contentM
             dstModel.BoundingBox = lodModelSrc.BoundingBox;
             dstModel.BoundingSphere = lodModelSrc.BoundingSphere;
 
-            /* 
-             * I think we dont need it, because can take the mats from the original model
-            * 
-            * foreach (var modelMaterial in lodModelSrc.Materials)
-            {
-                dstModel.Materials.Add(modelMaterial);
-            }*/
-
+            int meshId = 1;
             foreach (var mesh in lodModelSrc.Meshes)
             {
+                DateTime start = DateTime.Now;
```

Comment:
What is the expected time this remeshing will take? `DateTime.Now` is very imprecise, could be up to 60ms granularity. For much better higher precision, the `Stopwatch` should be used: https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=netstandard-2.0